docs/locale/ja/LC_MESSAGES/migration.po

```diff
@@ -96,7 +95,7 @@ msgstr "代わりに次の更新サイトをご利用ください。"
 
 #: ../../migration.rst:100
 msgid "https://spotbugs.github.io/eclipse/ (to use stable version, not ready yet)"
-msgstr "https://spotbugs.github.io/eclipse/ (安定バージョンを使用する (まだ利用できない))"
```

Comment:
Could you update English site in this PR? Then `msgid` for this translation can be updated.

Reply:
Yes.